src/liquid/components/ld-input/test/ld-input.spec.ts

```diff
@@ -371,18 +370,23 @@ describe('ld-input', () => {
   })
 
   it('removes hidden input field attributes', async () => {
-    const { root, waitForChanges } = await newSpecPage({
+    const page = await newSpecPage({
       components: [LdInput],
       html: `<form><ld-input dirname="example.dir" form="formName" name="example" value="hello" /></form>`,
     })
 
-    root.removeAttribute('dirname')
-    root.removeAttribute('form')
-    root.removeAttribute('value')
-    await waitForChanges()
+    const ldInput = page.body.querySelector('ld-input')
```

Comment:
I think page.root should work, as well, as the [Stencil docs](https://stenciljs.com/docs/unit-testing#spec-page-results) state:

> root - The first component found within the mocked document.body. If a component isn't found, then it'll return document.body.firstElementChild.

Reply:
Changed.